jobs/tests/test_search_jobs_view.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+from jobs.models import Job
+
+
+@pytest.mark.django_db
+def test_search_page_filtering_by_title_successfully_by_any_partial_title(client, user_HR, job_DataEngineer):
```

Comment:
I would remove `..search_page..` part from all the fixture names,
their names already long and it could be understood by the file name.
Something like `test_search_by_partial_job_title`